File: Leetcode/705.cs
using System.Collections;

public class MyHashSet
{
    private int MAX_LENGTH = 1000005;
    private BitArray mBitArray;
    /** Initialize your data structure here. */
    public MyHashSet()
    {
        mBitArray = new BitArray(MAX_LENGTH);
    }

    public void Add(int key)
    {
        mBitArray[key] = true;
    }

    public void Remove(int key)
    {
        mBitArray[key] = false;
    }

    /** Returns true if this set contains the specified element */
    public bool Contains(int key)
    {
        return mBitArray[key];
    }
}

File: Leetcode/705.cpp
class MyHashSet {
private:
    static const int MAX_LENGTH = 1000005;
    bitset<MAX_LENGTH> mBitset;
public:
    /** Initialize your data structure here. */
    MyHashSet() {

        
    }
    
    void add(int key) {
        mBitset[key] = true;
    }
    
    void remove(int key) {
        mBitset[key] = false;
    }
    
    /** Returns true if this set contains the specified element */
    bool contains(int key) {
        return mBitset[key];
    }
};

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet* obj = new MyHashSet();
 * obj->add(key);
 * obj->remove(key);
 * bool param_3 = obj->contains(key);
 */